recipe/meta.yaml

```diff
@@ -34,14 +37,24 @@ requirements:
     - scipy
     - pyproj
     - jinja2
+    {% if using_gdal %}
+    - gdal
+    {% endif %}
```

Comment:
from looking at the build artifacts, gdal is still being included as a dependency. Maybe some mistake on how this flag is being set? Most examples I've seen has use something more like: 

```
    - scipy
     - pyproj
     - jinja2
     - gdal  # [using_gdal]
```

Reply:
YAML's a hot mess, and it's probably getting passed through as a string. have tried with non-boolish values, will see what happens.

I just generally prefer jinja to selectors, the devil you know i guess.